utils/jq/Makefile

```diff
@@ -24,7 +24,8 @@ ifdef CONFIG_USE_MIPS16
   TARGET_CFLAGS += -fno-ipa-sra
 endif
 
-TARGET_CFLAGS += -std=c99
+# -D_GNU_SOURCE fixes https://github.com/stedolan/jq/issues/1598.
```

Comment:
Please don't leave comments in the Makefile

Reply:
@diizzyy Why not?

Reply:
@yousong 
The seems to be the general "rule" looking at previous discussions and in this case I don't see why the commit message shouldn't be enough.

Reply:
Sorry, I do not remember following such discussions.  Can you share some links to them?

In this specific case, jq the package does not work out of box and requires extra workaround whose cause and effect is not yet known and accepted into upstream project.  Instead of git-blaming for skimming through git-log, I think a plain comment surly serves better

Reply:
@yousong 
Unfortunately Google doesn't seem to index Github comments very well but it has been requested several times (not by me). It makes sense if it's a large Makefile or such but commit history makes it a lot neater and easier to read the code but I don't mind personally if people want to sprinkle comments everywhere.